web/pom.xml

```diff
@@ -45,23 +46,6 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
-    <!-- Azure dependencies -->
-    <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>applicationinsights-runtime-attach</artifactId>
-      <version>3.3.1</version>
-    </dependency>
-    <!-- Spring Cloud Azure dependencies -->
-    <dependency>
-      <groupId>com.azure.spring</groupId>
```

Comment:
If remove this dependency, can the sample works fine on Azure?

Reply:
As discussed in the scrum meeting, I recovered this.